### ISSUE TITLE
1.4.3 / 1.4. 11 Meeting contrast requirements via conforming alternate versions?

### ISSUE THREAD

**Issue by detlevhfischer** [w3c/wcag#3785]
The question is (or seems) simple.
Websites with text / graphics below the contrast requirements of 1.4.3 and 1.4.11 often offer contrast switches to load different styles that, after activation, increase contrast.

I assume that the option to meet contrast requirements via activating a fully accessible contrast switch is considered by many as sufficient to meet 1.4.3 / 1.4.11 (provided the mechanism is fully accessible and the resulting contrast of content rendered in the alternate version meets the requirements). 

**BUT:** 5.2.1 Conformance level requires that one of the (...) levels of conformance is **met in full**, which means for AA:
> For Level AA conformance, the Web page satisfies all the Level A and Level AA Success Criteria, or a Level AA conforming alternate version is provided."

This will often not be the case for a contrast switch that makes contrast conform but does not remedy other existing level AA lissues / FAILs of the page. Would this mean that a contrast switch can only be assumed to meet 1.4.3 / 1.4.11 if no other issues / FAILs remain (or those issues are miraculously remedied by the same switch)? Or is there a rationale besides "conforming alternate version" that would cover this contrast switch fallback practice?

**Comment by detlevhfischer** [comment]
@GreggVan Yes, it does - but I would appreciate a wider feedback indicating whether other auditors take the same line or would PASS pages that provide sufficient contrast via a switch **while** other FAILS exist that page (as is usually the case). My hunch is that this is a fairly common practice, but I may be wrong.

**Comment by mraccess77** [comment]
We have technique G174 https://www.w3.org/WAI/WCAG21/Techniques/general/G174

**Comment by fstrr** [comment]
This issue is labelled as a discussion, so we’re moving this to Discussions. There doesn’t seem to be an update to make to the documentation, but if that changes, we can move it back to the issues list.
